src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -82,12 +82,22 @@ public void put(Collection<SinkRecord> records) {
           sqle
       );
       String sqleAllMessages = "Exception chain:" + System.lineSeparator();
+      int totalExceptions = 0;
       for (Throwable e : sqle) {
         sqleAllMessages += e + System.lineSeparator();
+        totalExceptions++;
       }
       SQLException sqlAllMessagesException = new SQLException(sqleAllMessages);
       sqlAllMessagesException.setNextException(sqle);
       if (remainingRetries == 0) {
+        log.error(
+            "Failing task after exhausting retries; "
+              + "encountered {} exceptions on last write attempt",
+            totalExceptions);
+        int exceptionCount = 1;
+        for (Throwable e : sqle) {
+          log.error("Exception {}:", exceptionCount++, e);
```

Comment:
I understand that this is an error condition, but this log volume might be extreme for most cases. I think it's typical to run at INFO level, and drop down to debug for more info; so this might be a better fit for DEBUG severity.
```suggestion
          log.debug("Exception {}:", exceptionCount++, e);
```

Reply:
Fair, I've downgraded to `DEBUG` but added a note to the `ERROR`-level message that prompts users to enable `DEBUG` logging for more detail.